pkg/marathon/client.go

```diff
@@ -64,5 +65,31 @@ func (c *Client) GroupsWithoutRootSlash() (map[string]bool, error) {
 	default:
 		return nil, errors.New("unable to get Marathon groups")
 	}
+}
+
+// Info returns the content of the Marathon endpoint 'v2/info'.
+func (c *Client) Info() (map[string]interface{}, error) {
+	dcosClient := pluginutil.HTTPClient(c.baseURL)
+	resp, err := dcosClient.Get("/v2/info")
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	switch resp.StatusCode {
+	case 200:
+		data, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return nil, err
+		}
 
+		var result map[string]interface{}
+		err = json.Unmarshal(data, &result)
+		if err != nil {
+			return nil, err
+		}
+		return result, nil
+	default:
+		return nil, errors.New("unable to get Marathon info")
```

Comment:
How about adding first bytes of the response or at least status code. It's hard to tell what went wrong if you get `unable to get Marathon info` error.